Fix:

warning: variable 'maxy' set but not used [-Wunused-but-set-variable]

https://sourceforge.net/p/aewan/bugs/14/
--- bores/kurses.c
+++ bores/kurses.c
@@ -139,15 +139,11 @@ void kurses_color_at(int attr) {
 }
 
 int kurses_width() {
-   int maxy, maxx;
-   getmaxyx(stdscr, maxy, maxx);
-   return maxx;
+   return getmaxx(stdscr);
 }
 
 int kurses_height() {
-   int maxy, maxx;
-   getmaxyx(stdscr, maxy, maxx);
-   return maxy;
+   return getmaxy(stdscr);
 }
 
 int kurses_pos_valid(int x, int y) {
